src/visible.js

```diff
@@ -102,10 +102,6 @@ eg.module("visible", ["jQuery", eg, document], function($, ns, doc) {
 				delay = -1;
 			}
 
-			if (typeof delay === "undefined") {
-				delay = -1;
-			}
-
```

Comment:
According to jsdoc, I think below code is better:
`delay = Number.isInteger(delay) ? delay : -1;
containment = containment || false;`

Would you want me to pull request again?